### PR TITLE
NOISSUE - Fix opc-ua message type handling

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: opcua/api/transport.go [hunk 1]
```diff
@@ -24,6 +24,9 @@ const (
 
 	defOffset = 0
 	defLimit  = 10
+
+	defNamespace  = "ns=0"
+	defIdentifier = "i=84"
```
Comment (drasko) [inline opcua/api/transport.go]: Please leave a comment next to these magic numbers.

File: opcua/gopcua/browser.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/mainflux/mainflux/opcua"
 )
 
-const maxChildrens = 7
+const maxChildrens = 5
```
Comment (drasko) [inline opcua/gopcua/browser.go]: Same here - why 5, some comment please

**Review by drasko — CHANGES_REQUESTED** [review]

File: opcua/gopcua/browser.go [hunk 3]
```diff
@@ -180,31 +180,42 @@ func browse(n *opcuaGopcua.Node, path string, level int) ([]NodeDef, error) {
...
-
-		for _, rn := range refs {
-			children, err := browse(rn, def.Path, level+1)
-			if err != nil {
-				return err
-			}
-			nodes = append(nodes, children...)
-		}
-		return nil
+	bc, err := broseChildren(n, def.Path, level, id.HasComponent)
```
Comment (mteodor) [inline opcua/gopcua/browser.go]: `browse`

File: opcua/gopcua/browser.go (hunk 2 above)
Comment (mteodor) [reply opcua/gopcua/browser.go]: cant this be in env var

**Review by drasko — APPROVED** [review]
LGTM

File: opcua/gopcua/browser.go (hunk 2 above)
Comment (manuio) [reply opcua/gopcua/browser.go]: We will remove this and change the architecture. No need for now

**Pull request merged**